MicrosoftTeams.schema.json

```diff
@@ -455,6 +455,27 @@
                 "type": "string",
                 "maxLength": 2048
             }
+        },
+        "webApplicationInfo": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "string",
+                    "description": "AAD application id of the app. This id must be a GUID.",
+                    "maxLength": 36,
+                    "pattern": "^[0-9a-fA-F]{8}-([0-9a-fA-F]{4}-){3}[0-9a-fA-F]{12}$"
+                },
+                "resource": {
+                    "type": "string",
+                    "description": "Resource url of app for acquiring auth token for SSO.",
+                    "maxLength": 2048
+                }
+            },
+            "required": [
+                "id",
```

Comment:
How do we use this id?

Reply:
It will be used for contextual consent (a different feature that enables app access to team resources). We might end up using it too some day if we have to write user consent as part of the app install flow.